File: THCS/2022_2023/DongThap_2223/Bai1/LIENHOAN.py
import math

def main():
    # Đọc dữ liệu từ tệp LIENHOAN.INP
    with open("LIENHOAN.INP", "r") as f:
        n, k = map(int, f.readline().strip().split())

    # Tính số mặt cần chiên
    total_faces = 2 * n

    # Tính số lần chiên cần thiết
    batches = math.ceil(total_faces / k)

    # Thời gian tối thiểu
    min_time = batches * 3  # Mỗi lần chiên tốn 3 phút

    # Ghi kết quả vào tệp LIENHOAN.OUT
    with open("LIENHOAN.OUT", "w") as f:
        f.write(str(min_time) + "\n")

if __name__ == "__main__":
    main()

File: THCS/2022_2023/DongThap_2223/Bai1/LIENHOAN.cpp
#include <iostream>
#include <cmath> // Để sử dụng hàm ceil
#include <fstream> // Để đọc ghi tệp
using namespace std;

int main() {
    ifstream input("LIENHOAN.INP");
    ofstream output("LIENHOAN.OUT");
    
    int n, k;
    input >> n >> k; // Đọc số bánh và số bánh tối đa có thể chiên cùng lúc
    
    // Tính số mặt bánh cần chiên
    int total_faces = 2 * n;

    // Tính số lần chiên cần thiết
    int batches = (total_faces + k - 1) / k; // Tương đương với ceil(total_faces / k)

    // Thời gian tối thiểu
    int min_time = batches * 3; // Mỗi lần chiên tốn 3 phút

    output << min_time; // Ghi thời gian ra tệp
    input.close();
    output.close();

    return 0;
}
